adview-manager/serve/src/routes.rs

```diff
@@ -84,26 +89,26 @@ pub async fn get_preview_ad(Extension(state): Extension<Arc<State>>) -> Html<Str
         campaigns: vec![],
     };
 
-    // Mock the `get_market_demand_resp` call
+    // Mock the `get_units_for_slot_resp` call
     let mock_call = Mock::given(method("GET"))
-        // &depositAsset={}&depositAsset={}
+        // &depositAssets[]={}&depositAssets[]={}
         .and(path(format!("units-for-slot/{}", options.market_slot)))
-        // pubPrefix=HEX&depositAsset=0xASSET1&depositAsset=0xASSET2
+        // pubPrefix=HEX&depositAssets[]=0xASSET1&depositAssets[]=0xASSET2
         .and(query_param("pubPrefix", pub_prefix))
         .and(query_param(
-            "depositAsset",
+            "depositAssets[]",
             "0x6B175474E89094C44Da98b954EedeAC495271d0F",
         ))
-        // .and(query_param("depositAsset[]", "0x6B175474E89094C44Da98b954EedeAC495271d03"))
+        // .and(query_param("depositAssets[]", "0x6B175474E89094C44Da98b954EedeAC495271d03"))
```

Comment:
and this comment too. It was added before the units-for-slot was finished and that's why it was commented out. Now we have it above.